builtin/tag.c

```diff
@@ -103,9 +104,15 @@ static int delete_tag(const char *name, const char *ref,
 }
 
 static int verify_tag(const char *name, const char *ref,
-				const unsigned char *sha1)
+				const unsigned char *sha1, const char *fmt_pretty)
```

Comment:
Junio suggests to replace `const char *fmt_pretty` with `void *cb_data` in the signature and then do
`const char *fmt_pretty = cb_data;` in the function body.

But can't we just leave it be? I.e. wouldn't it be okay to have `fn(*p, ref, sha1, cb_data)` call 
```C
static int verify_tag(const char *name, const char *ref,
 				const unsigned char *sha1)	const unsigned char *sha1, const char *fmt_pretty)
```